ProQuest.js

```diff
@@ -386,7 +404,7 @@ function scrape(doc, url, type, pdfUrl) {
 		}
 	}
 
-	item.url = url;
+	item.url = url.replace(/\?accountid=\d+/, '');
```

Comment:
don't include the `?`, since it would break the URL if there are any other parameters.
`/\baccountid=[^&#]+&?/`